Not sure why this test is failing at the moment:

```
======================================================================
ERROR: corehq.apps.userreports.tests.test_data_source_partitioning:DataSourcePartitionByOwnerLastChars.test_partitioned_by_date
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_data_source_partitioning.py", line 155, in test_partitioned_by_date
    self._process_docs([sample_doc1, sample_doc2, sample_doc3])
  File "/mnt/commcare-hq-ro/corehq/apps/userreports/tests/test_data_source_partitioning.py", line 44, in _process_docs
    pillow = get_case_pillow(ucr_configs=[self.data_source])
  File "/mnt/commcare-hq-ro/corehq/pillows/case.py", line 104, in get_case_pillow
    checkpoint = KafkaPillowCheckpoint(checkpoint_id, topics)
  File "/mnt/commcare-hq-ro/corehq/ex-submodules/pillowtop/checkpoints/manager.py", line 173, in __init__
    self._get_checkpoints()
  File "/mnt/commcare-hq-ro/corehq/ex-submodules/pillowtop/checkpoints/manager.py", line 176, in _get_checkpoints
    return KafkaCheckpoint.get_or_create_for_checkpoint_id(self.checkpoint_id, self.topics)
  File "/mnt/commcare-hq-ro/corehq/ex-submodules/pillowtop/models.py", line 93, in get_or_create_for_checkpoint_id
    .values_list('topic', 'partition').count()
  File "/vendor/lib/python2.7/site-packages/django/db/models/query.py", line 364, in count
    return self.query.get_count(using=self.db)
  File "/vendor/lib/python2.7/site-packages/django/db/models/sql/query.py", line 499, in get_count
    number = obj.get_aggregation(using, ['__count'])['__count']
  File "/vendor/lib/python2.7/site-packages/django/db/models/sql/query.py", line 480, in get_aggregation
    result = compiler.execute_sql(SINGLE)
  File "/vendor/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 887, in execute_sql
    cursor = self.connection.cursor()
  File "/vendor/lib/python2.7/site-packages/django/db/backends/base/base.py", line 254, in cursor
    return self._cursor()
  File "/vendor/lib/python2.7/site-packages/django/db/backends/base/base.py", line 231, in _cursor
    return self._prepare_cursor(self.create_cursor(name))
  File "/vendor/lib/python2.7/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/vendor/lib/python2.7/site-packages/django/db/backends/base/base.py", line 231, in _cursor
    return self._prepare_cursor(self.create_cursor(name))
  File "/vendor/lib/python2.7/site-packages/django/db/backends/postgresql/base.py", line 220, in create_cursor
    cursor = self.connection.cursor()
InterfaceError: connection already closed
----------------------------------------------------------------------
```

--- a/docker/hq-compose-citus.yml
+++ b/docker/hq-compose-citus.yml
@@ -15,6 +15,7 @@ services:
     depends_on: { citus_master: { condition: service_healthy } }
     environment:
       CITUS_HOST: citus_master
+      POSTGRES_DB: commcare_ucr_citus
   citus_worker1:
     image: 'citusdata/citus:8.1.1'
     ports: ["${CITUS_WORKER1_EXTERNAL_PORT:-5601}:5432"]

--- a/docker/hq-compose-citus.yml
+++ b/docker/hq-compose-citus.yml
@@ -7,6 +7,8 @@ services:
     labels: ['com.citusdata.role=Master']
     volumes:
       - ${VOLUME_PREFIX}citus_master:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: commcare_ucr_citus
   citus_manager:
     image: 'citusdata/membership-manager:0.2.0'
     volumes: ['/var/run/docker.sock:/var/run/docker.sock']
@@ -20,6 +22,8 @@ services:
     depends_on: { citus_manager: { condition: service_healthy } }
     volumes:
       - ${VOLUME_PREFIX}citus_worker1:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: commcare_ucr_citus
   citus_worker2:
     image: 'citusdata/citus:8.1.1'
     ports: ["${CITUS_WORKER2_EXTERNAL_PORT:-5602}:5432"]
@@ -27,3 +31,5 @@ services:
     depends_on: { citus_manager: { condition: service_healthy } }
     volumes:
       - ${VOLUME_PREFIX}citus_worker2:/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: commcare_ucr_citus

--- a/docker/hq-compose.yml
+++ b/docker/hq-compose.yml
@@ -111,6 +111,7 @@ services:
     depends_on: { citus_master: { condition: service_healthy } }
     environment:
       CITUS_HOST: citus_master
+      POSTGRES_DB: commcare_ucr_citus
   citus_worker1:
     image: 'citusdata/citus:8.1.1'
     labels: ['com.citusdata.role=Worker']

--- a/docker/hq-compose.yml
+++ b/docker/hq-compose.yml
@@ -103,6 +103,8 @@ services:
     labels: ['com.citusdata.role=Master']
     volumes:
       - ${VOLUME_PREFIX}${BLANK_IF_TESTS-citus_master:}/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: commcare_ucr_citus
   citus_manager:
     image: 'citusdata/membership-manager:0.2.0'
     volumes: ['/var/run/docker.sock:/var/run/docker.sock']
@@ -115,9 +117,13 @@ services:
     depends_on: { citus_manager: { condition: service_healthy } }
     volumes:
       - ${VOLUME_PREFIX}${BLANK_IF_TESTS-citus_worker1:}/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: commcare_ucr_citus
   citus_worker2:
     image: 'citusdata/citus:8.1.1'
     labels: ['com.citusdata.role=Worker']
     depends_on: { citus_manager: { condition: service_healthy } }
     volumes:
       - ${VOLUME_PREFIX}${BLANK_IF_TESTS-citus_worker2:}/var/lib/postgresql/data
+    environment:
+      POSTGRES_DB: commcare_ucr_citus

--- a/docker/localsettings.py
+++ b/docker/localsettings.py
@@ -85,8 +85,6 @@ if USE_PARTITIONED_DATABASE:
 
     WAREHOUSE_DATABASE_ALIAS = 'warehouse'
 
-# CitusDB setup is done by Django migrations in the testapps.citus_* apps. These
-# migrations are only run during unit tests.
 DATABASES.update({
     'icds-ucr': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
@@ -98,37 +96,7 @@ DATABASES.update({
         'PORT': '5432',
         'TEST': {
             'SERIALIZE': False,
-            'DEPENDENCIES': ['citus-ucr-worker1', 'citus-ucr-worker2'],
         },
-        'ROLE': 'citus_master'
-    },
-    'citus-ucr-worker1': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'DISABLE_SERVER_SIDE_CURSORS': True,
-        'NAME': 'commcare_ucr_citus',
-        'USER': 'postgres',
-        'PASSWORD': '',
-        'HOST': 'citus_worker1',
-        'PORT': '5432',
-        'MIGRATE': False,
-        'TEST': {
-            'SERIALIZE': False,
-        },
-        'ROLE': 'citus_worker'
-    },
-    'citus-ucr-worker2': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'DISABLE_SERVER_SIDE_CURSORS': True,
-        'NAME': 'commcare_ucr_citus',
-        'USER': 'postgres',
-        'PASSWORD': '',
-        'HOST': 'citus_worker2',
-        'PORT': '5432',
-        'MIGRATE': False,
-        'TEST': {
-            'SERIALIZE': False,
-        },
-        'ROLE': 'citus_worker'
     },
 })
 

--- a/docker/localsettings.py
+++ b/docker/localsettings.py
@@ -110,6 +110,7 @@ DATABASES.update({
         'PASSWORD': '',
         'HOST': 'citus_worker1',
         'PORT': '5432',
+        'MIGRATE': False,
         'TEST': {
             'SERIALIZE': False,
         },
@@ -123,6 +124,7 @@ DATABASES.update({
         'PASSWORD': '',
         'HOST': 'citus_worker2',
         'PORT': '5432',
+        'MIGRATE': False,
         'TEST': {
             'SERIALIZE': False,
         },

